metrics/internal/collectors/common_test.go

```diff
@@ -0,0 +1,63 @@
+package collectors
```

Comment:
rename file to `testutil.go`. `*_test.go` will be run as test files.

Reply:
ack